billboardd/tklockinterface.h

```diff
@@ -0,0 +1,62 @@
+
+/**
+ * Billboard - Low Power Mode Standby Screen for the N9
+ * Webpage: http://thp.io/2012/billboard/
+ * Copyright (C) 2012, 2013, 2014 Thomas Perl <thp.io/about>
+ * Copyright 2015 Elliot Wolk <elliot.wolk@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ **/
+
+#ifndef TKLOCKINTERFACE_H
+#define TKLOCKINTERFACE_H
+
+#include <QObject>
+
+class TklockInterface : public QObject
+{
+    Q_OBJECT
+
+    public:
+        TklockInterface();
+        ~TklockInterface();
+
+        bool isLocked() {
```

Comment:
now that we dont have use of a display string, isLocked() is a more useful api to provide, as it doesnt require knowing the tklock modes and provides the single useful bit of information about the state.